Loop/Views/SettingsViewModel.swift

```diff
@@ -49,6 +49,8 @@ public class SettingsViewModel: ObservableObject {
 
     @ObservedObject var pumpManagerSettingsViewModel: DeviceViewModel
     @ObservedObject var cgmManagerSettingsViewModel: DeviceViewModel
+    
+    lazy private var trash = Set<AnyCancellable>()
```

Comment:
`cancellables` is better here; the purpose of storing them is exactly the opposite of throwing them in the trash!

Reply:
You mean as a variable name? Sure thing.